### PR TITLE
gh-extension: add list and remove examples

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/gh-extension.md [hunk 1]
```diff
@@ -22,3 +22,15 @@
...
 
 `gh extension upgrade --all`
+
+- List installed extensions:
+
+`gh extension list`
+
+- Remove a extension:
+
+`gh extension remove {{name}}`
```
Comment (CleanMachine1) [inline pages/common/gh-extension.md:32]:
```suggestion
`gh extension remove {{extension_name}}`
```

**Review by CleanMachine1 — APPROVED** [review]
lgtm after comment is addressed

File: pages/common/gh-extension.md [hunk 2]
```diff
@@ -22,3 +22,15 @@
 - Upgrade all extensions:
 
 `gh extension upgrade --all`
+
+- List installed extensions:
+
+`gh extension list`
+
+- Remove a extension:
```
Comment (marchersimon) [inline pages/common/gh-extension.md:30]:
```suggestion
- Remove an extension:
```

File: pages/common/gh-extension.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/gh-extension.md:32]: The 1st and 4rd examples use the same `{{name}}` placeholder.

Comment (marchersimon) [reply pages/common/gh-extension.md:32]: I think we should change those too.

**Review by pixelcmtd — APPROVED** [review]

File: pages/common/gh-extension.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/gh-extension.md:32]: Most of the time we`d just use `{{extention}}`, like we do for e.g. `{{package}}`. What do you think about that?

Comment (sbrl) [reply pages/common/gh-extension.md:32]: `{{extension_name}}` or `{{extension}}` would work fine here.

**Review by marchersimon — APPROVED** [review]
Thanks @navarroaxel

**Pull request merged**